src/explore-education-statistics-admin/src/pages/release/data/components/ReorderList.module.scss

```diff
@@ -1,72 +1,50 @@
 @import '~govuk-frontend/govuk/base';
 
-.dropArea {
-  background: govuk-colour('light-grey');
-  list-style: none;
-  margin: 0 govuk-spacing(2) govuk-spacing(2) 0;
-  padding: 0;
-}
-
-.dropAreaActive {
-  outline: $govuk-focus-width solid $govuk-focus-colour;
-}
-
-.draggable {
-  background: govuk-colour('white');
-  border-bottom: 1px solid govuk-colour('dark-grey');
-  margin-bottom: 0;
-  padding: 0 0 0 govuk-spacing(7);
-  position: relative;
+.isExpandedItem {
+  background: govuk-colour('light-grey') !important;
 
   &::before {
-    color: govuk-colour('dark-grey');
-    content: '\2630';
-    cursor: grab;
-    font-size: 1.6rem;
-    left: 4px;
-    position: absolute;
-    top: 7px;
-  }
-
-  &:focus,
-  &.isDragging {
-    @include govuk-focused-text;
-  }
-
-  &.isDraggedOutside {
-    opacity: 0.7;
-  }
-}
-
-.draggable.isDisabled {
-  &::before {
-    content: none;
-  }
-}
-
-.draggable.isExpanded {
-  background: govuk-colour('light-grey');
-
-  &::before {
-    content: '\25BC';
+    content: '▼' !important;
     font-size: 1rem;
-    left: 8px;
-    top: 12px;
+    left: 12px;
+    top: 14px;
   }
 
-  .isExpanded {
+  .isExpandedItem {
     background: lighten(govuk-colour('light-grey'), 2);
   }
 }
 
 .draggableInner {
+  align-items: center;
   display: flex;
   flex-grow: 1;
   justify-content: space-between;
-  padding: govuk-spacing(2) govuk-spacing(2) govuk-spacing(2) 0;
+  padding: 0 govuk-spacing(2) 0 0;
 }
+
 .optionLabel {
   &.isExpanded {
     font-weight: $govuk-font-weight-bold;
+    padding-left: 40px;
+  }
+
+  &.hideIcon {
+    padding-left: 40px;
+  }
+}
+
+.inner {
+  width: 100%;
+}
+
+.dropArea {
+  margin-bottom: govuk-spacing(2);
+  .dropArea & {
+    margin: govuk-spacing(2) govuk-spacing(1) 0 0;
```

Comment:
Similar to above, things are a bit tight here. Would suggest increasing the bottom margin to something like spacing 4 so we get:

![image](https://user-images.githubusercontent.com/9917868/208936486-5c4d6529-f5c6-4f9e-91d5-c97001f620f4.png)